hive-metastore/src/main/java/org/apache/iceberg/hive/legacy/LegacyHiveTableUtils.java

```diff
@@ -59,9 +59,9 @@ static Schema getSchema(org.apache.hadoop.hive.metastore.api.Table table) {
     org.apache.avro.Schema avroSchema = schemaStr != null ? new org.apache.avro.Schema.Parser().parse(schemaStr) : null;
     Schema schema;
     if (avroSchema != null) {
-      FileFormat format = serdeToFileFormat(table.getSd().getSerdeInfo().getSerializationLib());
+      String serde = table.getSd().getSerdeInfo().getSerializationLib();
       org.apache.avro.Schema finalAvroSchema;
-      if (format.equals(FileFormat.AVRO) || HasDuplicateLowercaseColumnNames.visit(avroSchema)) {
+      if (serde.equals("org.apache.hadoop.hive.serde2.avro.AvroSerDe") || HasDuplicateLowercaseColumnNames.visit(avroSchema)) {
```

Comment:
Could this be defined as constant? I notice "org.apache.hadoop.hive.serde2.avro.AvroSerDe" also shows up in other places.

Reply:
I think it's fine to use it as is, since there are not other code re-use it.